src/libraries/System.Formats.Tar/src/System/Formats/Tar/PaxTarEntry.cs

```diff
@@ -119,5 +150,29 @@ public PaxTarEntry(TarEntryType entryType, string entryName, IEnumerable<KeyValu
 
         // Determines if the current instance's entry type supports setting a data stream.
         internal override bool IsDataStreamSetterSupported() => EntryType == TarEntryType.RegularFile;
+
+        // Checks if the extended attributes dictionary contains 'atime' and 'ctime'. If any of them is not found, it is added with the value of either the current entry's 'mtime', or 'DateTimeOffset.UtcNow', depending on the value of 'useMTime'.
```

Comment:
This comment is quite long (longer than my widescreen monitor is wide), please chop it into a couple lines.  e.g.

```suggestion
        // Checks if the extended attributes dictionary contains 'atime' and 'ctime'.
        // If any of them is not found, it is added with the value of either the current entry's 'mtime',
        // or 'DateTimeOffset.UtcNow', depending on the value of 'useMTime'.
```

Reply:
Fixed.

Reply:
Ideally all of the long lines would have gotten trimmed, but I guess I'll take what I can get :smile:.

My personal cutoff is somewhere around 120 characters on a line, since that's about when GitHub's file view requires a scrollbar (unless you use a style plugin to make the file view wider... but I like to cater to the "average" person looking at the code)